modules/.submodules.json

```diff
@@ -11,13 +11,15 @@
     "criteoIdSystem",
     "netIdSystem",
     "identityLinkIdSystem",
-    "sharedIdSystem"
+    "sharedIdSystem",
+    "haloIdSystem"
   ],
   "adpod": [
     "freeWheelAdserverVideo",
     "dfpAdServerVideo"
   ],
   "rtdModule": [
-    "browsiRtdProvider"
+    "browsiRtdProvider",
+    "audigentRtdProvider"
```

Comment:
should this be in a separate PR since it seems unrelated to the user id module submission?

Reply:
the user id submodule and rtd probider are going to be linked in a future pr